app/pathogen/arbovirus/analyze/page.tsx

```diff
@@ -8,34 +8,53 @@ import {
 import Filters from "@/app/pathogen/arbovirus/dashboard/filters";
 import ArboDataTable from "@/app/pathogen/arbovirus/analyze/ArboDataTable";
 import { ScrollArea } from "@/components/ui/scroll-area";
+import { AntibodyPathogenBar, MedianSeroPrevByWHOregion, MedianSeroPrevByWHOregionAndAgeGroup, StudyCountOverTime, StudyCountOverTimeBySampleFrame, Top10CountriesByPathogenStudyCount, WHORegionAndArbovirusBar } from "./recharts";
+import clsx from "clsx";
 
 const VisualizationCard = (props: {
   title: string;
   children: React.ReactNode;
+  height?: string;
 }) => {
+  
+  
   return (
-    <Card className={"mb-4 mr-4"}>
-      <CardHeader>
-        <CardTitle>{props.title}</CardTitle>
-      </CardHeader>
-      <CardContent className={"px-0 h-72"}>{props.children}</CardContent>
+    <Card className={clsx("mb-4 mr-4 p-4 pt-0", props.height ?? "h-full sm:h-3/4 2xl:h-1/2 relative")}>
+      <CardContent className={"px-0 h-full flex flex-col"}>
+        <h3 className="py-4 w-full text-center text-lg">{props.title}</h3>
+        {props.children}
+      </CardContent>
     </Card>
   );
 };
 
 export default function ArboAnalyze() {
   return (
     <>
-      <div className={"col-span-5 row-span-2 flex flex-col overflow-auto"}>
-        <VisualizationCard title={"Count Antibody Pathogen - Client"}>
-          <CountOfStudiesStratifiedByAntibodyAndPathogen />
+      <div className={"col-span-5 row-span-2 overflow-auto"}>
+        <VisualizationCard title={"Estimate count by WHO region and pathogen"}>
+          <WHORegionAndArbovirusBar />
+        </VisualizationCard>
+        <VisualizationCard title={"Estimate count by arbovirus & antibody type"}>
+          <AntibodyPathogenBar />
+        </VisualizationCard>
+        <VisualizationCard title={"Median seroprevalence by WHO Region"} height="h-full">
+          <MedianSeroPrevByWHOregion />
```

Comment:
For WHO region charts, can we remove estimates where WHO region is N/A? (Leave them on the map for now, but we may have to change this later for our WHO partnership) 

Also can we force the x-axis label to display all WHO regions?
<img width="202" alt="Screen Shot 2023-12-04 at 9 59 00 AM" src="https://github.com/PathoTracker/Pathotracker/assets/76851774/493bc064-2217-4d3b-8f36-33a0b764ac6e">